MQ2Medley.cpp

```diff
@@ -46,6 +86,7 @@ MQ2Medley
 /medley delay # - 10ths of a second, minimum of 0, default 3, how long after casting a spell to wait to cast next spell
 /medley reload - reload the INI file
 /medley quiet - Toggles songs listing for medley and queued songs
+/medley bandolier - Toggles between using MQ2Exchange or In-Game Bandolier
```

Comment:
Are you using exchange?  I consider it an obsolete plugin I wouldn't want to encourage it's use.  Maybe needed for emu?

Reply:
Yeah using exchange, i chose that for its simplicity and that it is already loaded in a lot of macros. I thought about using MQ2BardSwap but that is just using exchange itself.

Reply:
you are also not limited to just mainhand offhand swaps, you can swap and pair many item|slot pairs in the swapset, 
this was to account for any augs or items not primary secondary slot.